rust/lanzatool/src/install.rs

```diff
@@ -8,37 +9,71 @@ use nix::unistd::sync;
 use tempfile::tempdir;
 
 use crate::esp::EspPaths;
-use crate::generation::Generation;
+use crate::gc::Roots;
+use crate::generation::{Generation, GenerationLink};
 use crate::pe;
 use crate::signature::KeyPair;
 
 pub struct Installer {
+    gc_roots: RefCell<Roots>,
```

Comment:
It would be helpful to add a comment why `RefCell` is necessary here. Did you lose against the borrow checker? :)

Reply:
I thought interior mutability would be the right pattern here, so I don't have to pass the entire struct around as mutable. But, since RefCell also incurs a runtime penality making the entire struct mutable is probably better.